src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,73 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are three mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- Seconding
+  - A proposal is second'ed when a team member formally endorses the proposal. It is intended that
+    seconding only occur once discussion has concluded and team members have had time to raise
+    concerns. Seconding tentatively accepts a proposal subject to a ten-day waiting period for
+    other team members to raise any concerns.
+  - Seconding can only be used to approve a MCP.
```

Comment:
There are no rustdoc MCPs at the time of writing, so this section should be removed.

I know there have been some informal proposals thrown around about unifying certain processes like MCPs across all teams but since there's no official proposal or even sign of acceptance, this isn't something that can be considered serious yet.

Reply:
Arf, I missed this one.